pkg/provider/googleapps/googleapps.go

```diff
@@ -737,3 +744,11 @@ func isAppId(val string) string {
 	}
 	return appId
 }
+
+func extractDevicePushExtraNumber(doc *goquery.Document) string {
+	extraNumber := ""
+	doc.Find("div[jsname=feLNVc]").Each(func(_ int, s *goquery.Selection) {
```

Comment:
I am wondering if it is possible for this to change? It looks like a generated name, which might  be different from page to page?

Reply:
I agree that this is a generated name, and therefore it's less than ideal to use. But it seems to me that this is the best we can do. 

I checked in an anonymized version of the HTML at `pkg/provider/googleapps/example/challenge-extra-number.html` –– as you can see, if we don't want to use generated names, we have to couple ourselves to the HTML structure, which is not ideal either.

A couple of mitigating circumstances that make me feel better about the usage of jsname:
1. Elsewhere in this file we have `func extractKeyHandles` that relies on a jsname as well.
2. [@martin-shippit's comment on Aug 31 '22](https://github.com/Versent/saml2aws/pull/777#issuecomment-1233520239) indicates that this logic worked for him too, so his page must have had the same jsname.

Reply:
OK, thanks for the explanation.